lib/cWarnings.ml

```diff
@@ -54,6 +54,7 @@ let set_category_status ~name status =
 
 let is_all_keyword name = CString.equal name "all"
 let is_none_keyword s = CString.equal s "none"
+let is_current_keyword s = CString.equal s "current"
```

Comment:
This `current` stuff seems like it should be a different PR with changelog and doc.

Reply:
ok, I'll try to do that.